### PR TITLE
API tests for CRUD operations

### PR DESCRIPTION
**Pull request by kpazniak13**

### CODE REVIEW

File: cypress/e2e/api_testing/gorest_api_delete_tests.cy.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+describe('WHen I test all the PUT operations on Gorest User table', () => {
```
Comment (AndreiRymskikh) [inline cypress/e2e/api_testing/gorest_api_delete_tests.cy.js]: What do you mean "PUT"? All the request have DELETE method here

File: cypress/e2e/api_testing/gorest_api_delete_tests.cy.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    it('Then deleting with all the fields of the created user works correctly', () => {
+        cy.request({
+            method: 'DELETE',
+            auth: {
+                bearer: Cypress.env('bearerToken')
+            },
+            url: `https://gorest.co.in/public/v2/users/${userId}`,
+        })
+        .then(response => {
+            expect(response.status).to.equal(204);
```
Comment (AndreiRymskikh) [inline cypress/e2e/api_testing/gorest_api_delete_tests.cy.js:53]: Is it possible also to check was it deleted or not?

**Review by AndreiRymskikh — CHANGES_REQUESTED** [review]

File: cypress/e2e/api_testing/gorest_api_delete_tests.cy.js (hunk 1 above)
Comment (kpazniak13) [reply cypress/e2e/api_testing/gorest_api_delete_tests.cy.js]: fixed

File: cypress/e2e/api_testing/gorest_api_delete_tests.cy.js (hunk 2 above)
Comment (kpazniak13) [reply cypress/e2e/api_testing/gorest_api_delete_tests.cy.js:53]: added assertion

**Review by AndreiRymskikh — APPROVED** [review]

**Pull request merged**